README.md

```diff
@@ -1,6 +1,6 @@
 [![pipeline status](https://gitlab.com/cscs-ci/ci-testing/webhook-ci/mirrors/4700071344751697/7514005670787789/badges/master/pipeline.svg)](https://gitlab.com/cscs-ci/ci-testing/webhook-ci/mirrors/4700071344751697/7514005670787789/-/commits/master) [![codecov](https://codecov.io/gh/eth-cscs/DLA-Future/branch/master/graph/badge.svg)](https://codecov.io/gh/eth-cscs/DLA-Future)
 
-# Distributed Linear Algebra with Futures.
+# Distributed Linear Algebra for the Future
```

Comment:
Possible alternatives:

```suggestion
# Distributed Linear Algebra from the Future
```
as in "from the future, but _for the present_".

```suggestion
# DLA-Future
```
as in "Future" used to have a meaning, but now it's "just a name".

```suggestion
# DLAF
```
in the same spirit as the previous but this would probably require renaming the repo to DLAF as well (and the spack package etc.).